administrator/components/com_fields/helpers/fields.php

```diff
@@ -170,8 +173,23 @@ public static function getFields ($context, $item = null, $prepareValue = false)
 					{
 						try
 						{
+							/*
+							 *On before field render
+							 *Event allow plugins to modfify the output of the field before it is rendered
+							 */
+                            $dispatcher = JEventDispatcher::getInstance();
+							if($field->value)
+								$dispatcher->trigger('onFieldBeforeRender',array($context,'mustache',&$field));
```

Comment:
The renderer context parameter was added as I thought it would be a nice to give the developer the option to respond differently if the rendering is done via Mustache i.e. by the end user or via  the component or template override. This is not essential, but I thought it could be useful in some cases. 

If you want that parameter removed, I am happy to remove that parameter.